src/jit/gentree.h

```diff
@@ -479,8 +479,8 @@ struct GenTree
     // happening.
     void CopyCosts(const GenTree* const tree)
     {
-        INDEBUG(gtCostsInitialized =
-                    tree->gtCostsInitialized;) // If the 'tree' costs aren't initialized, we'll hit an assert below.
+        INDEBUG(gtCostsInitialized = tree->gtCostsInitialized;) // If the 'tree' costs aren't initialized, we'll hit an
+                                                                // assert below.
```

Comment:
Again, not sure why this changed - but if it's going to change, the comment should be moved above.

Reply:
Hmm, on my computer, jit-format + vs code is not stable on a few jit source files, which can output different format with each run, especially for some comments or whitespace. 
Will fix, sorry for the mess.